app/main/forms/suppliers.py

```diff
@@ -252,8 +252,16 @@ class CompanyTradingStatusForm(FlaskForm):
         },
     ]
 
-    trading_status = DMRadioField(
+    trading_status = RadioField(
         "What’s your trading status?",
-        hint="This information will be used to find out about the types of companies on frameworks.",
         validators=[InputRequired(message="You must choose a trading status.")],
-        options=OPTIONS)
+        choices=[(option["value"], option["label"]) for option in OPTIONS],
+        id="input-trading_status-1"  # TODO: change to input-reuse when on govuk-frontend~3
+    )
+
+    def get_items(self):
+        return [
+            {'value': value, 'text': text, 'checked': checked}
+            for (value, text, checked)
+            in self.trading_status.iter_choices()
+        ]
```

Comment:
In other places (for instance [briefs frontend](https://github.com/alphagov/digitalmarketplace-briefs-frontend/blob/master/app/main/forms/award_or_cancel.py)) we've added the items as an attribute on the field in the form constructor, this is perfectly reasonable too though.

Reply:
I think the difference between this and your example is that we need to handle `checked`, which tracks which radio choice is pre-checked. As far as Alex and I could tell, `iter_choices` is the only thing that gives us that information, hence this helper method being necessary.

We're planning to revisit when we come to do the other two radios in this journey, when it should be clearer what the right way to do this is.